pyp_kcwi/sub_sky.py

```diff
@@ -0,0 +1,142 @@
+# This function is used to subtract sky median
+#
+# INPUT:     - sky_file
+#            - img_file
+#            - mask_limit    mask limit
+#            - cut_ch        cut channel number
+#
+# INPUT EXAMPLE:
+#   median('kb171021_00083_icuber.fits', 'kb171021_00082_icuber.fits', 40, cut_ch=300)
+#
+# MODIFICATION HISTORY:
+#   Written by: Qiong Li
+#   2017-11-01 Initial version
+
+def median(sky_file, img_file, mask_limit, cut_ch=300):
```

Comment:
Add a doc string describing what this method is intended to do.

Reply:
Ah, I see now that the doc string is above.

Please put it below the def
using Python syntax.

Reply:
Done.